NebulaModel/Packets/Players/NewDroneOrderPacket.cs

```diff
@@ -8,16 +11,18 @@ public class NewDroneOrderPacket
         public ushort PlayerId { get; set; }
         public int Stage { get; set; }
         public int Priority { get; set; }
+        public Float3 EntityPos { get; set; }
         
         public NewDroneOrderPacket() { }
-        public NewDroneOrderPacket(int planetId, int droneId, int entityId, ushort playerId, int stage, int priority)
+        public NewDroneOrderPacket(int planetId, int droneId, int entityId, ushort playerId, int stage, int priority, Vector3 entityPos)
         {
             PlanetId = planetId;
             DroneId = droneId;
             EntityId = entityId;
             PlayerId = playerId;
             Stage = stage;
             Priority = priority;
+            EntityPos = new Float3(entityPos);
```

Comment:
Should we be consistent and use `.ToFloat3()` instead?

Reply:
fixed :)